Source/SwiftyJSON.swift

```diff
@@ -1376,3 +1376,30 @@ func >=(lhs: NSNumber, rhs: NSNumber) -> Bool {
         return lhs.compare(rhs) != NSComparisonResult.OrderedAscending
     }
 }
+
+
+//MARK: - Save and load JSON locally using NSUserDefaults.
+extension JSON {
+    public func saveToLocal(forKey: String) -> Bool {
```

Comment:
This is a good use case. Can we make the function name more readable by doing so ```saveToLocal``` -> ```saveLocally``` and the same for loadLocally